deploy/backplane/OWNERS

```diff
@@ -0,0 +1,6 @@
+reviewers:
+- srep-functional-leads
+- srep-team-leads
+approvers:
+- srep-team-leads
+- srep-architects
```

Comment:
```suggestion
- srep-architects
options:
  no_parent_owners: true
```

You need ^^ to actually restrict parent owners

Reply:
:bowing_man: